### PR TITLE
DPMMA-2815 Campaign stats in preview graph

### PR DESCRIPTION
**Pull request by patrykgruszka**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🟢
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 

## Description
This PR adds additional info to the campaign preview graph:
- Decisions (success/failed percent, completed, pending)
- Actions (completed, pending)
- Conditions (success/failed percent, completed, pending)
image

---
### 📋 Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Add campaign with some actions, conditions and decisions
3. Execute the campaign
```
ddev exec php bin/console mautic:campaign:update 
ddev exec php bin/console mautic:campaign:trigger
```
4. Check if the stats in preview are correct

### CODE REVIEW

File: app/bundles/CampaignBundle/Resources/views/Event/_preview.html.twig [hunk 1]
```diff
@@ -39,6 +49,12 @@ data-event-id="{{ event['id'] }}">
                 <span class="fs-18 fw-l">{{ ('mautic.campaign.'~event['type'])|trans }}</span>
                 <span class="campaign-event-name ellipsis">{% if ('dev' == constant('MAUTIC_ENV')) %}<small>{{ event['id'] }}</small> {% endif %}{{ event['name'] }}</span>
             </div>
+            <div>
+                <span class="label label-success" data-toggle="tooltip" title="{{ 'mautic.report.campaign.completed.actions'|trans }}"><span class="ri-user-6-fill"></span> {{ event.logCountProcessed }}</span>
+                {% if (event.logCountForPending) %}
+                <span class="label label-default" data-toggle="tooltip" title="{{ 'mautic.report.campaign.pending.actions'|trans }}"><span class="ri-hourglass-line"></span> {{ event.logCountForPending }}</span>
+                {% endif %}
+            </div>
```
Comment (andersonjeccel) [inline app/bundles/CampaignBundle/Resources/views/Event/_preview.html.twig]:
```suggestion
            <div class="campaign-event-stat">
                <span data-toggle="tooltip" title="{{ 'mautic.report.campaign.completed.actions'|trans }}"><span class="ri-check-line"></span> {{ event.logCountProcessed }}</span>
                {% if (event.logCountForPending) %}
                <span data-toggle="tooltip" title="{{ 'mautic.report.campaign.pending.actions'|trans }}">| <span class="ri-hourglass-line"></span> {{ event.logCountForPending }}</span>
                {% endif %}
            </div>
```

image

This change could make the presentation cleaner to users

File: app/bundles/CampaignBundle/Assets/css/campaign.css [hunk 2]
```diff
@@ -107,6 +107,18 @@
...
+    background-color: var(--support-success);
+    color: var(--text-inverse);
+}
+
+.jtk-label--error {
+    border-color: var(--support-error);
+    background-color: var(--support-error);
+    color: var(--text-inverse);
+}
+
```
Comment (andersonjeccel) [inline app/bundles/CampaignBundle/Assets/css/campaign.css:121]:
```suggestion

.campaign-event-stat {
    position: absolute;
    bottom: 0;
    right: 0;
    padding: 4px 8px;
}
```

The CSS styling for my changes in the Twig template

**Review by andersonjeccel — CHANGES_REQUESTED** [review]
The functionality is awesome, it's nice to see how much we can move the builder forward with only a few lines of code

I suggested some styling fixes (screenshot attached)

File: app/bundles/CampaignBundle/Resources/views/Event/_preview.html.twig (hunk 1 above)
Comment (patrykgruszka) [reply app/bundles/CampaignBundle/Resources/views/Event/_preview.html.twig]: It looks nice, but what about the conditions and decisions? Should we change them to keep the statistics consistent? Do you have any idea where to place it?

Comment (andersonjeccel) [reply app/bundles/CampaignBundle/Resources/views/Event/_preview.html.twig]: Oh, the idea was to apply this to all events
Now I see that my suggestion targeted only one type

**Review by tomekkowalczyk — APPROVED** [review]
I think it's a very cool feature, as for the code, I have no comments

File: app/bundles/CampaignBundle/Resources/views/Event/_preview.html.twig (hunk 1 above)
Comment (patrykgruszka) [reply app/bundles/CampaignBundle/Resources/views/Event/_preview.html.twig]: Yes, and the layout in conditions and decisions is a bit different:
image

Comment (patrykgruszka) [reply app/bundles/CampaignBundle/Resources/views/Event/_preview.html.twig]: Maybe something like this? Also there could appear second stat - pending
image

Comment (andersonjeccel) [reply app/bundles/CampaignBundle/Resources/views/Event/_preview.html.twig]: The main goal would be to use an absolute position for the stats, to take less space and also be great when the pending number appears
So from the first image, I would just place the stat in the right location within the template

What do you think about this idea?

Comment (patrykgruszka) [reply app/bundles/CampaignBundle/Resources/views/Event/_preview.html.twig]: IMO only the ideal case scenario with action looks good. Even events with longer names will overlap the counters. Have a look
image

Comment (andersonjeccel) [reply app/bundles/CampaignBundle/Resources/views/Event/_preview.html.twig]: Right, let's go with the last working solution 

image

Comment (andersonjeccel) [reply app/bundles/CampaignBundle/Resources/views/Event/_preview.html.twig]: @patrykgruszka tell me when it's ready, I'll do the user testing

Comment (patrykgruszka) [reply app/bundles/CampaignBundle/Resources/views/Event/_preview.html.twig]: @andersonjeccel thanks! It's ready to test.

**Review by andersonjeccel — APPROVED** [review]
It works, thank you for the improvement

image

**Pull request merged**